### PR TITLE
Update to current running code

### PR DESCRIPTION
**Pull request by scfc**

### CODE REVIEW

**Comment by nemobis** [comment]
Thanks! I saw things happening on the project, are these changes you already applied on the live code?

File: Common.css [hunk 1]
```diff
@@ -1,25 +1,25 @@
-@import "//bits.wikimedia.org/static-current/skins/MonoBook/main.css";
-@import "//bits.wikimedia.org/static-current/resources/src/mediawiki.legacy/shared.css";
+@import "//en.wikipedia.org/w/skins/MonoBook/main.css";
+@import "//en.wikipedia.org/w/resources/src/mediawiki.legacy/shared.css";
```
Comment (nemobis) [inline Common.css:2]: This reminds me I wanted to switch some URLs to Meta.

File: persoonsgegevens/inc/errorhandler.inc.php [hunk 2]
```diff
@@ -1,102 +1,102 @@
...
-				error_log($error_msg, 3, $log_file);
-			}
-		}
-	}
-
-	if($halt_script)
-	{
+        $error_msg = $errstr . ' in ' . $errfile . ' on ' . $errline . ' at ' . date("r") . '.';
+        $error_msg_html = $errstr . ' in <b>' . $errfile . '</b> on line <b>' . $errline . '</b> at ' . date("r") . '.';
+        $email_addr = 'erwin85@hemlock.ts.wikimedia.org';
```
Comment (nemobis) [inline persoonsgegevens/inc/errorhandler.inc.php:6]: Is this causing problems? Should we switch it to tools.erwin85@tools.wmflabs.org?

File: xc/index.php [hunk 3]
```diff
@@ -1,14 +0,0 @@
-<html>
-<head>
-</head>
-<body>
-<?php
-$files = scandir('./img');
-foreach($files as $f) {
-    if (substr($f, -4) == '.png') {
-        echo '<img src="./img/' . $f . '">';
```
Comment (nemobis) [inline xc/index.php:9]: Gotta check .gitignore too

**Review by nemobis — APPROVED** [review]

**Comment by nemobis** [comment]
Thanks a lot again

File: Common.css (hunk 1 above)
Comment (scfc) [reply Common.css:2]: JFTR: These `bits.wikimedia.org` had been done by someone else in the working directory (you?), I just committed them.

File: persoonsgegevens/inc/errorhandler.inc.php (hunk 2 above)
Comment (scfc) [reply persoonsgegevens/inc/errorhandler.inc.php:6]: AFAICS `$email = False;` prevents that code from ever being executed.  Personally I prefer errors to go to `error.log` than being mailed every three seconds :-), so this is one of several code paths I would just remove.

**Pull request merged**